src/scripts/modules/search/search-template.html

```diff
@@ -1 +1 @@
-<div class="search"></div>
+<div class="search" aria-busy="true"></div>
```

Comment:
If the AJAX request never starts (because of a bug in the JS code) then it seems like this indicator is incorrect. Could it instead be added when `@model = searchResults.config().load({query: location.search})` is called?

Accurately adding the attribute when the AJAX request is sent may help with testing (to find bugs when the search never began)